packages/11ty/package.json

```
@@ -6,8 +6,9 @@
   "main": ".eleventy.js",
   "scripts": {
     "build": "ELEVENTY_ENV=production eleventy",
-    "build:pdf": "npx pagedjs-cli@latest ${ELEVENTY_OUTPUT_DIR:-_site}/pdf.html",
-    "clean": "del ${ELEVENTY_OUTPUT_DIR:-_site} public/pdf.html",
+    "build:pdf": "pagedjs-cli public/pdf.html --outline-tags 'h1' -o paged.pdf",
+    "build:prince": "prince public/pdf.html --output prince.pdf",
+    "clean": "del ${ELEVENTY_OUTPUT_DIR:-_site} public",
```

In case a user puts something else in the `public` directory we can use a glob pattern here
```suggestion
    "clean": "del ${ELEVENTY_OUTPUT_DIR:-_site} public/*.pdf",
```

---

Ah great, updated. (I also added a CSS glob)